src/java/org/apache/cassandra/audit/AuditLogOptions.java

```diff
@@ -58,4 +61,11 @@
      * For more options, refer: net.openhft.chronicle.queue.RollCycles
      */
     public String roll_cycle = "HOURLY";
+
+    public String toString()
```

Comment:
Is this the `toString()` implementation we want? Like the other three, it's missing fields, intentionally or not (if intentionally, maybe comment why so?). Also, this string helper prefixes the map with the class name, and it doesn't look pretty in CQL output, subjectively.